api/serializers.py

```diff
@@ -70,7 +70,7 @@ def create(self, validated_data):
                 raise ValueError('Collection only accepts items of type ' + collection_type)
 
         status = 'pending'
-        if user.has_perm(['api.approve_collection_items', 'api.approve_meeting_items'], collection) or allow_all:
+        if user.has_perms(['api.approve_collection_items', 'api.approve_meeting_items'], collection) or allow_all:
```

Comment:
I think this needs to be `user.has_perm('api.approve_collection_items', collection)` or `user.has_perm('api.approve_meeting_items', collection)